src/utils/http.py

```diff
@@ -83,6 +83,8 @@ def check_url_contains_pdf(url) -> bool:
     if url is None:
         return False
     try:
+        if "sciencedirect" in url and "download=false" in url:
```

Comment:
special clause here -- the sciencedirect doesn't work with the scraper, but the link to their pdf works